src/v2/guide/forms.md

```diff
@@ -15,7 +15,7 @@ Bạn có thể sử dụng directive `v-model` để tạo ràng buộc dữ li
 ### Văn bản
 
 ``` html
-<input v-model="message" placeholder="edit me">
+<input v-model="message" placeholder="Nhập văn bản">
```

Comment:
Shouldn't it be "Nhập thông điệp" instead?

Reply:
Someone corrected it. 😄

Reply:
Corrected it. Please, re-check